### PR TITLE
i added a message in the userlogin screen of "your cart isn't empty" …

### PR DESCRIPTION
**Pull request by eilonasher**
…if the cart isn't empty so the user can edit the items if he wants instead of invite them again. for that I made Some changes to add a text view and function in carts to identify if cart is empty. please make your notes

### CODE REVIEW

File: app/src/main/java/com/example/sabonit/Cart.java [hunk 1]
```diff
@@ -85,4 +85,13 @@ public void setOrdersList(ArrayList<Order> ordersList) {
...
     }
 
+    /**
+     * check if the order list is empty or not
+     * @return empty - true, not empty - false
+     */
+    public boolean isCartEmpty()
+    {
+        return this.ordersList.size() == 0;
+    }
```
Comment (galpinto25) [inline app/src/main/java/com/example/sabonit/Cart.java:95]: OK, great function to use, simple and nice that you added the Javadoc to it

File: app/src/main/java/com/example/sabonit/CartIsntEmptyDialogFragment.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        frag.setArguments(args);
+        return frag;
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.your_cart_isnt_empty, container);
+    }
+
```
Comment (galpinto25) [inline app/src/main/java/com/example/sabonit/CartIsntEmptyDialogFragment.java:33]: Looks OK

File: app/src/main/java/com/example/sabonit/UserLoginActivity.java [hunk 3]
```diff
@@ -119,6 +121,7 @@ public void onComplete(@NonNull Task<DocumentSnapshot> task) {
                         Log.d("SearchAccountIdFailed", "No such document");
                         writeNewAccount();
                     }
+                    displayMessageIfCartIsEmpty();
```
Comment (galpinto25) [inline app/src/main/java/com/example/sabonit/UserLoginActivity.java:124]: Great, now the user is informed if its cart is not empty from previous entries

File: app/src/main/java/com/example/sabonit/UserLoginActivity.java [hunk 4]
```diff
@@ -187,4 +190,15 @@ public void popHowItWorks(View view) {
...
+    {
+        if (! Account.getCurrentAccount().getCart().isCartEmpty())
+        {
+            FragmentManager fm = getSupportFragmentManager();
+            CartIsntEmptyDialogFragment cartIsntEmptyDialogFragment = CartIsntEmptyDialogFragment.newInstance();
+            cartIsntEmptyDialogFragment.show(fm, "cart_isnt_empty");
+        }
+
+    }
+
```
Comment (galpinto25) [inline app/src/main/java/com/example/sabonit/UserLoginActivity.java:203]: Looks OK!

File: app/src/main/java/com/example/sabonit/Cart.java (hunk 1 above)
Comment (eilonasher) [reply app/src/main/java/com/example/sabonit/Cart.java:95]: thanks!

File: app/src/main/java/com/example/sabonit/CartIsntEmptyDialogFragment.java (hunk 2 above)
Comment (eilonasher) [reply app/src/main/java/com/example/sabonit/CartIsntEmptyDialogFragment.java:33]: thank you for the help to write it!

File: app/src/main/java/com/example/sabonit/UserLoginActivity.java (hunk 3 above)
Comment (eilonasher) [reply app/src/main/java/com/example/sabonit/UserLoginActivity.java:124]: yes like what we talked about

**Pull request merged**